composer.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "psr/log-util",
+  "description": "Utilities for the psr/log package that backs the PSR-3 specification.",
+  "minimum-stability": "dev",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "PHP-FIG",
+      "homepage": "https://www.php-fig.org/",
+      "role": "Organisation"
+    },
+    {
+      "name": "Anton Ukhanev",
+      "email": "xedin.unknown@gmail.com",
+      "role": "Developer"
+    }
+  ],
+  "require": {
+    "php": "^5.5 | ^7.0 | ^8.0",
+    "psr/log": "^1.1.4"
+  },
+  "require-dev": {
+    "phpunit/phpunit": "^4.0 | ^5.0 | ^6.0 | ^7.0 | ^8.0 | ^9.0",
```

Comment:
AFAIK, this is the _only_ way that it will work, because you need PHPUnit 4 to support PHP 5.4 and 5.5 at least. And it absolutely does work, as evidenced by CI and what I explain in https://github.com/php-fig/log/pull/75#issuecomment-855381201.

Reply:
I know that PHPUnit 4 is needed. I'm just not sure there's an easy way to make it work with such a wide constraint.

Point is, once you've declared a wide constraint and tagged it, we can't take it back, so we will not tag it if we can't assure it works.

Actions are not starting as it is, we will merge this and do subsequent PRs to fix stuff later.

Reply:
I imagine, we will remove PHPUnit versions prior to 8 from the 2.0 (or 0.2, depending on how you want to take this) version of this package, because legacy compat will not be a problem. And in 1.x there's probably no need to remove them, as they are required for legacy PHP versions, as you agreed.